inrupt/pod-browser
# Feat: privacy page

# Feat: privacy page

- Add a Privacy page which displays contacts (people) and apps
  - I decided to keep the Contacts page as it is today and create a new Privacy page which essentially displays the contacts as well. This is to make the the feature flag logic simpler, and to start migrating from contacts to audit list more gradually, but this page reuses a lot of the Contacts page components and functionality for now.
  - For now, the only app displayed is hardcoded in the AgentList (a single Thing created just for the purpose of having something to display)

- Add the privacy page to feature flag

- Make Tabs more generic so they can be reused throughout the app

- Add tests

Note: Prism version needs to be updated to a proper patch before merging

## To test:

- Verify that you can see the Privacy tab with a WebID included in the feature flag
- Verify that you cannot see the Privacy tab with a WebID not included in the feature flag
- Verify that Contacts page works as usual

# Checklist
